tools/gen_selectedops.py

```diff
@@ -1,80 +1,99 @@
-import os
-import sys
+#!/usr/bin/env python3
+
+import argparse
+import pathlib
 
 OPMAP_TO_CMAKE_FLAGS = {
-    'BlingFireSentenceBreaker': 'OCOS_ENABLE_BLINGFIRE',
-    'GPT2Tokenizer': 'OCOS_ENABLE_GPT2_TOKENIZER',
-    'WordpieceTokenizer': 'OCOS_ENABLE_WORDPIECE_TOKENIZER',
-    'StringConcat': 'OCOS_ENABLE_TF_STRING',
-    'StringECMARegexReplace': 'OCOS_ENABLE_TF_STRING',
-    'StringECMARegexSplitWithOffsets': 'OCOS_ENABLE_TF_STRING',
-    'StringEqual': 'OCOS_ENABLE_TF_STRING',
-    'StringToHashBucket': 'OCOS_ENABLE_TF_STRING',
-    'StringToHashBucketFast': 'OCOS_ENABLE_TF_STRING',
-    'StringJoin': 'OCOS_ENABLE_TF_STRING',
-    'StringLength': 'OCOS_ENABLE_TF_STRING',
-    'StringLower': 'OCOS_ENABLE_TF_STRING',
-    'StringRegexReplace': 'OCOS_ENABLE_RE2_REGEX',
-    'StringRegexSplitWithOffsets': 'OCOS_ENABLE_RE2_REGEX',
-    'StringSplit': 'OCOS_ENABLE_TF_STRING',
-    'StringToVector': 'OCOS_ENABLE_TF_STRING',
-    'StringUpper': 'OCOS_ENABLE_TF_STRING',
-    'SegmentExtraction': 'OCOS_ENABLE_MATH',
-    'StringMapping': 'OCOS_ENABLE_TF_STRING',
-    'VectorToString': 'OCOS_ENABLE_TF_STRING',
-    'MaskedFill': 'OCOS_ENABLE_TF_STRING',
-    'BertTokenizer': 'OCOS_ENABLE_BERT_TOKENIZER',
-    'BasicTokenizer': 'OCOS_ENABLE_BERT_TOKENIZER',
-    'BertTokenizerDecoder': 'OCOS_ENABLE_BERT_TOKENIZER',
-    'SentencepieceTokenizer': 'OCOS_ENABLE_SPM_TOKENIZER',
-    'ImageDecode': 'OCOS_ENABLE_VISION',
-    'ImageEncode': 'OCOS_ENABLE_VISION',
+    "BlingFireSentenceBreaker": "OCOS_ENABLE_BLINGFIRE",
+    "GPT2Tokenizer": "OCOS_ENABLE_GPT2_TOKENIZER",
+    "WordpieceTokenizer": "OCOS_ENABLE_WORDPIECE_TOKENIZER",
+    "StringConcat": "OCOS_ENABLE_TF_STRING",
+    "StringECMARegexReplace": "OCOS_ENABLE_TF_STRING",
+    "StringECMARegexSplitWithOffsets": "OCOS_ENABLE_TF_STRING",
+    "StringEqual": "OCOS_ENABLE_TF_STRING",
+    "StringToHashBucket": "OCOS_ENABLE_TF_STRING",
+    "StringToHashBucketFast": "OCOS_ENABLE_TF_STRING",
+    "StringJoin": "OCOS_ENABLE_TF_STRING",
+    "StringLength": "OCOS_ENABLE_TF_STRING",
+    "StringLower": "OCOS_ENABLE_TF_STRING",
+    "StringRegexReplace": "OCOS_ENABLE_RE2_REGEX",
+    "StringRegexSplitWithOffsets": "OCOS_ENABLE_RE2_REGEX",
+    "StringSplit": "OCOS_ENABLE_TF_STRING",
+    "StringToVector": "OCOS_ENABLE_TF_STRING",
+    "StringUpper": "OCOS_ENABLE_TF_STRING",
+    "SegmentExtraction": "OCOS_ENABLE_MATH",
+    "StringMapping": "OCOS_ENABLE_TF_STRING",
+    "VectorToString": "OCOS_ENABLE_TF_STRING",
+    "MaskedFill": "OCOS_ENABLE_TF_STRING",
+    "BertTokenizer": "OCOS_ENABLE_BERT_TOKENIZER",
+    "BasicTokenizer": "OCOS_ENABLE_BERT_TOKENIZER",
+    "BertTokenizerDecoder": "OCOS_ENABLE_BERT_TOKENIZER",
+    "SentencepieceTokenizer": "OCOS_ENABLE_SPM_TOKENIZER",
+    "ImageDecode": "OCOS_ENABLE_VISION",
+    "ImageEncode": "OCOS_ENABLE_VISION",
 }
 
 
-def gen_cmake_oplist(opconfig_file, oplist_cmake_file='_selectedoplist.cmake'):
```

Comment:
This file name was fixed to be consistent with cmake option OCOS_ENABLE_SELECTED_OPLIST which will include the file name. It's good to use this option to build the package, but I didn't find how you specify the file name on CMake configuration.

Reply:
the path actually passed to this is still the same. the default argument value was always being overridden before so I just removed it
https://github.com/microsoft/onnxruntime-extensions/blob/56ef48073c4e58c44c69f27e9b5fb7c8cf29b7f4/tools/gen_selectedops.py#L93-L99